### ISSUE TITLE
Warning: A non-numeric value encountered in /extensions/SemanticComments/specials/Comment/CE_CommentParserFunctions.php on line 379

### ISSUE THREAD

**Issue by ZoXx** [krabina/Datencockpit#43]
Hallo zusammen,
bei den Paragraphen habe ich folgenden Fehler oben über der Header Bar:
Warning: A non-numeric value encountered in /extensions/SemanticComments/specials/Comment/CE_CommentParserFunctions.php on line 379

kann mir da wer weiterhelfen?

**Comment by ZoXx** [comment]
Nochmal nachgeschaut. 
Also es geht um den Bereich:
`foreach ( $queryResult as $res ) {
	$sum += $res;
}`
Habe ihn auskommentiert und nun tritt der Fehler natürlich nicht mehr auf. Habt ihr hier was geändert?

**Comment by jifflaender** [comment]
Moin @ZoXx ,
wenn ich Paragraphen aus dem BDSG aufrufe, sehe ich eine solche Fehlermeldung nicht. Kannst Du den Link posten, bei dem der Fehler erscheint (ab ...\index.php)?

**Comment by krabina** [comment]
Die Semantic Comments extension ist leider nicht gut maintained. Geändert haben wir sicher nichts, Sie wurde für das Projekt gefixt. Könnte sein, dass das ein Bug in der Extenion ist. Ich gehe dazu über, die Semantic Comments extension gegen die aktuellere CommentStreams zu tauschen:
https://www.mediawiki.org/wiki/Extension:CommentStreams

Ich würde daher empfehlen, die internen Kommentarfunktion vorerst nicht zu nutzen, da beim Upgrade eine Datencockpits auf V 1.31LTS die Semantic Comments extension dann nicht mehr funktionieren wird. Daher wäre es (auch jetzt schon) besser, CommentStreams zu verwenden, das sollte auch mit MW 1.27 laufen.

**Comment by ZoXx** [comment]
@krabina kann man die Kommentarfunktion abschalten?

@jifflaender bei mi rist der Fehler bei allen Paragraphen z.B. 
index.php?title=DSGVO:Artikel_7._Bedingungen_für_die_Einwilligung

**Comment by jifflaender** [comment]
Moin @ZoXx ,

Ah, das sind die Artikel (DSGVO), nicht die Paragraphen (BDSG). Dennoch, bei mir taucht der Fehler auch da nicht auf. Allerdings wird die Kommentarfunktion auch nicht genutzt. Wenn Du Dein System komplett neu aufgesetzt hast, solltest Du ja auch keine Kommentare haben. Von daher: strange, aber ich kann den Fehler leider nicht nachstellen...

**Comment by ZoXx** [comment]
Ist bei den Artikeln der DSGVO und des BDSG. RIchtig, die sind alle weg, trotzdem ist der Fehler da. Habe es erstmal auskommentiert, war die Zählfunktion
Viele Grüße

**Comment by krabina** [comment]
> @krabina kann man die Kommentarfunktion abschalten?

ja, in den Vorlagen nach {{ShowComments|show=True}} suchen und das entfernen. Alternativ/zusätzlich SemanticComments in Localsettings.php auskommentieren.
